systest/Makefile

```diff
@@ -93,8 +93,8 @@ TESTENVLOGDIR := $(RESULTS_DIR)/f5-openstack-agent_$(SANITIZED_BRANCH)/testenv
 #    Tests use tox for python variants.
 setup:
 	@echo executing $@
-	sudo -E -H $(SCRIPT_DIR)/install_test_infra.sh ;\
-	mkdir -p $(TESTENVLOGDIR)
```

Comment:
Dependencies are now rolled into the `agenttestrunner` I a full test run will be necessary to land the above... but significantly the smoke test does pass.